tasks/test.py

```diff
@@ -92,11 +121,128 @@ def install_tools(ctx):
                     ctx.run(f"go install {tool}")
 
 
-@task()
+def lint_common(ctx, modules: List[GoModule], fail_on_fmt: bool):
```

Comment:
I would love to have a `for_each(modules: List[GoModule], callback: Callable[GoModule, *Args], *args)` function to unify these, but maybe it's too much of a hassle.

Reply:
I can add a todo there (and a card in our backlog) to do this.